extensions/community_extensions/Protected Audience Support.md

```diff
@@ -0,0 +1,153 @@
+# Bid Request Values
+
+## Object: InterestGroupAuctionSupport 
```

Comment:
@hillslatt, missing here is the context - that this `InterestGroupAuctionSupport` object is proposed as a new attribute, as I recall `igs` on `Imp.ext` (in PROVISIONAL it was in `Imp` _proper_). Ditto for the response Objects.